src/category.rs

```diff
@@ -161,6 +162,24 @@ impl Category {
         ).bind::<Text, _>(&self.category)
             .load(conn)
     }
+
+    pub fn parent_categories(&self, conn: &PgConnection) -> QueryResult<Vec<Category>> {
```

Comment:
Could you add a doc comment here outlining what this function does??

Reply:
You know I thought I had done that but I forgot about the three-slashes thing. Sorry about that.